src/components/Pager.vue

```diff
@@ -0,0 +1,374 @@
+<!--
+  This source file is part of the Swift.org open source project
+
+  Copyright (c) 2024 Apple Inc. and the Swift project authors
+  Licensed under Apache License v2.0 with Runtime Library Exception
+
+  See https://swift.org/LICENSE.txt for license information
+  See https://swift.org/CONTRIBUTORS.txt for Swift project authors
+-->
+<template>
+  <div class="pager">
+    <template v-if="pages.length === 1">
+      <slot name="page" :page="pages[0]" />
+    </template>
+    <template v-else>
+      <div class="container">
+        <Gutter class="left">
+          <ControlPrevious
+            :disabled="!hasPreviousPage"
+            @click.native="previous"
+          />
+        </Gutter>
+        <div class="viewport" ref="viewport">
```

Comment:
![Screenshot 2024-03-01 at 17 25 43](https://github.com/apple/swift-docc-render/assets/8567677/a6541c07-417d-45c4-a347-d8d5d3a70bc8)

The focus outline of items gets cut because of the overflow property in this element, we may want to fix that by adding a padding to it.

Reply:
Interesting. Not exactly sure what might be causing this off the top of my head—will need to dig in and try to see if I can see why this happens.

Reply:
I don't think this is a blocker at all.

Reply:
Didn't find it yet. Agreed it's not a blocker but would be nice to look into in the future.